testsuite/model/src/test/java/org/keycloak/testsuite/model/parameters/MultiSiteInfinispan.java

```diff
@@ -31,7 +29,7 @@
 /**
  * @author <a href="mailto:mkanis@redhat.com">Martin Kanis</a>
  */
-public class CrossDCInfinispan extends KeycloakModelParameters {
+public class MultiSiteInfinispan extends KeycloakModelParameters {
```

Comment:
can be removed as `org.keycloak.testsuite.model.parameters.RemoteInfinispan` already creates everything required for multi-site.